Sources/NIOWebSocket/WebSocketFrameDecoder.swift

```diff
@@ -221,7 +221,7 @@ public final class WebSocketFrameDecoder: ByteToMessageDecoder {
     public var cumulationBuffer: ByteBuffer? = nil
 
     /// The maximum frame size the decoder is willing to tolerate from the remote peer.
-    private let maxFrameSize: Int
+    public let maxFrameSize: Int
```

Comment:
we can also keep this internal if we like

Reply:
my vote would go to let's keep it internal :)

Reply:
Yup, I think we should keep it internal too.

Reply:
It's always easier to make things more public than to make them less public.